corputils/models.py

```diff
@@ -153,7 +164,7 @@ def portrait_url(self, size=32):
     def get_member_objects(self, user):
         show_apis = self.show_apis(user)
         return sorted([CorpStats.MemberObject(id, name, show_apis=show_apis) for id, name in self.members.items()],
-                      key=attrgetter('character_name'))
+                      key=attrgetter('main_user'))
```

Comment:
You should leave `'character_name'` as the second argument for `attrgetter` otherwise the sorting for unaffiliated characters gets rather bad. e.g. `key=attrgetter('main_user', 'character_name')`